src/app/zap_cluster_list.json

```diff
@@ -182,6 +183,7 @@
         "LOW_POWER_CLUSTER": ["low-power-server"],
         "MEDIA_INPUT_CLUSTER": ["media-input-server"],
         "MEDIA_PLAYBACK_CLUSTER": ["media-playback-server"],
+        "MICROWAVE_OVEN_CONTROL_CLUSTER": ["microwave-oven-control-server"],
```

Comment:
Not sure we need these pieces until we are implementing either...  But if we do, ok.